x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/rest/inference/RestStartTrainedModelDeploymentAction.java

```diff
@@ -87,107 +82,45 @@ protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient
         }
 
         if (restRequest.hasParam(TIMEOUT.getPreferredName())) {
-            TimeValue openTimeout = validateParameters(
-                request.getTimeout(),
-                restRequest.paramAsTime(TIMEOUT.getPreferredName(), StartTrainedModelDeploymentAction.DEFAULT_TIMEOUT),
-                StartTrainedModelDeploymentAction.DEFAULT_TIMEOUT
-            ); // hasParam, so never default
-            request.setTimeout(openTimeout);
+            request.setTimeout(restRequest.paramAsTime(TIMEOUT.getPreferredName(), request.getTimeout()));
```

Comment:
I'm passing the `request.get*()` as the default because it'll be defaulted for each value (except the cache size) when `StartTrainedModelDeploymentAction.Request` is initialized either when parsing the body or in the case of no body.